src/causalprog/algorithms/moments.py

```diff
@@ -20,7 +21,12 @@ def sample(
     keys = jax.random.split(rng_key, len(nodes))
 
     for node, key in zip(nodes, keys, strict=False):
-        values[node.label] = node.sample(values, samples, rng_key=key)
+        values[node.label] = node.sample(
+            {} if parameter_values is None else parameter_values,
```

Comment:
```suggestion
            parameter_values if parameter_values else {},
```

Depending on how you prefer your Python syntax-tic sugar